src/app/(auth)/signup/page.tsx

```diff
@@ -66,6 +78,7 @@ export default function SignUp() {
           'https://immigration-justice-project.vercel.app/email-verified',
```

Comment:
do we want to use CONFIG here too? i.e. 
`https://immigration-justice-project.vercel.app/{CONFIG.emailverified}`

Reply:
hm, should we add the whole link to CONFIG?

Reply:
Good point; Since this is the only place that we would be routing to the reset-password page (ie we’ll never need a route to ‘/reset-password’), I’m down for adding the entire link to CONFIG!

Reply:
added!